LeetCode

korostik opened the pull request.

korostik committed:
обновления

korostik committed:
не проходит на 1566 тесте...

korostik committed:
Delete 2.AddTwoNumbers.cpp

korostik committed:
не проходит на 1566 тесте

korostik committed:
2 have completed but 143 haven not completed yet

korostik committed:
longest palindrome with 2 letters

korostik committed:
done

korostik committed:
done

korostik committed:
done

korostik committed:
theme:Prefix Sum

korostik committed:
theme:prefix sum 1413, 1422

korostik committed:
one of the most important task about subarray

korostik committed:
Update 209.MinimumSizeSubarraySum.py

korostik committed:
prefix sum

korostik committed:
prefix sum

korostik committed:
work on Linked List

korostik committed:
work on Linked List

korostik committed:
Find the smallest substring in s that contains all characters of t (including duplicates), or return "" if none exists.

korostik committed:
Find the smallest substring in s that contains all characters of t (including duplicates), or return "" if none exists.

korostik committed:
Find the smallest substring in s that contains all characters of t (including duplicates), or return "" if none exists.

korostik committed:
Daily Task. Count the number of hills (peaks with smaller neighbors) and valleys (dips with larger neighbors) in an array, treating consecutive duplicates as a single element.

korostik committed:
delete 2210

korostik committed:
delete Readme.md and individual

korostik committed:
Count elements that are higher or lower than their neighbors.

justas05 reviewed (commented):
А почему мы удалили файлы из individual?

korostik committed:
sum in a[left:right + 1] in tree

korostik committed:
ATTEMTED
For every fruit  choose the most left, suitable basket.
The answer  is number of fruits which we can`t put in basket

korostik committed:
ATTEMTED
max in list[left:right] using a segment tree

korostik committed:
ATTEMTED
For every fruit  choose the most left, suitable basket.
The answer  is number of fruits which we can`t put in basket

korostik committed:
ATTEMTED
For every fruit  choose the most left, suitable basket.
The answer  is number of fruits which we can`t put in basket

korostik committed:
Given a positive integer n represented as a minimal sorted array of powers of 2, and a 2D array queries, return an array where each element is the product of elements in powers from left_i to right_i (modulo 10^9 + 7).

justas05 commented:
Давай мы разнесем эти изменения по нескольким ПРам

justas05 commented on leetcode/147.InsertionSortList.cpp:
```diff
@@ -26,7 +26,7 @@ class Solution {
 private:
     ListNode* replace_elements(ListNode* temp, ListNode* &node, int k) {
-        if (k == -1) { 
+        if (k == -1) {
```
Это стилистические правки, их можно выделить в отдельный ПР

korostik commented:
При rebase в ветке LeetCode появилась папка с файлами individual, поэтому мы ее удалили.----------------
                        Кому: korostik/Task ***@***.***);
                        Копия: Author ***@***.***);
                        Тема: [korostik/Task] LeetCode (PR #25);
                        30.07.2025, 08:39, "Andrew Makarovskiy" ***@***.***>:
                        
@justas05 commented on this pull request.

А почему мы удалили файлы из individual?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

korostik committed:
delete some useless things

justas05 reviewed (changes requested):
Так я же уже несколько раз говорил, что надо разбить на несколько ПРов.

korostik commented:
А, сейчас.----------------
                        Кому: korostik/Task ***@***.***);
                        Копия: Author ***@***.***);
                        Тема: [korostik/Task] LeetCode (PR #25);
                        20.08.2025, 16:17, "Andrew Makarovskiy" ***@***.***>:
                        
@justas05 requested changes on this pull request.

Так я же уже несколько раз говорил, что надо разбить на несколько ПРов.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
